app/build.gradle

```diff
@@ -28,8 +28,8 @@ android {
         applicationId "crux.bphc.cms"
         minSdkVersion 24
         targetSdkVersion 29
-        versionCode 1080500
-        versionName "1.8.5"
+        versionCode 1080600
+        versionName "1.8.6"
```

Comment:
Again, you don't need to bump version in this commit. This will be done in a separate commit, which will then be tagged using `git-tag`.